manifest.json

```diff
@@ -51,6 +51,7 @@
         "contextMenus",
         "webRequest",
         "webRequestBlocking",
-        "*://*.duckduckgo.com/"
+        "*://*.duckduckgo.com/",
+        "notifications"
```

Comment:
No, it does not. It only adds a line that says `Display notifications` to the description:
![screenshot from 2016-09-28 00-33-57](https://cloud.githubusercontent.com/assets/461491/18894453/4ef39a2a-8513-11e6-815f-441d85d3f4f6.png)